scripts/cluster

```diff
@@ -24,7 +28,63 @@ function createGenesis() {
     --premine 0x85da99c8a7c2c95964c8efd687e95e632fc533d6:1000000000000000000000 \
     --premine 0x0000000000000000000000000000000000000000 \
     --epoch-size 10 \
-    --burn-contract 0:0x0000000000000000000000000000000000000000
+    --burn-contract 0:0x0000000000000000000000000000000000000000 \
+    --reward-wallet 0xDEADBEEF:1000000
+}
+
+function initRootchain() {
+  echo "Initializing rootchain"
+
+  ./polygon-edge rootchain server > /dev/null &
+  sleep 5
```

Comment:
Could we ping `http://localhost:8545` (JSON RPC address), instead of waiting predefined amount of time?

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1502/commits/9ef5987e2b371a44a07abc5dee60695a7899d7bd